package.json

```diff
@@ -12,12 +12,14 @@
     "@polkadot/ui-keyring": "^0.42.1",
     "@polkadot/util": "^1.2.1",
     "@polkadot/util-crypto": "^1.2.1",
+    "big.js": "^5.2.2",
```

Comment:
Dropping this link in case anyone else is curious about big.js like I was. I agree with your choice to use it.

[What is the difference between big.js, bignumber.js and decimal.js?](https://github.com/MikeMcl/big.js/wiki)